src/main/java/net/sf/mzmine/modules/rawdatamethods/rawdataimport/fileformats/ZipReadTask.java

```diff
@@ -189,6 +192,8 @@ public String getTaskDescription() {
     public void cancel() {
         if (copy != null)
             copy.cancel();
+        //Is it possible to delete the temporary files here?  
+        //Otherwise canceling the task potential leaves an incompletely decompressed file
```

Comment:
Yes that is what I observed, but if there is no quitting before trying to reload a file, an exception is thrown as the existing cached .mzML is incomplete